pkg/util/runtimeconfig/manager.go

```diff
@@ -44,20 +45,26 @@ type Manager struct {
 	config    interface{}
 
 	configLoadSuccess prometheus.Gauge
+	configInfo        *prometheus.GaugeVec
 }
 
 // NewRuntimeConfigManager creates an instance of Manager and starts reload config loop based on config
 func NewRuntimeConfigManager(cfg ManagerConfig, registerer prometheus.Registerer) (*Manager, error) {
 	mgr := Manager{
 		cfg: cfg,
 		configLoadSuccess: prometheus.NewGauge(prometheus.GaugeOpts{
-			Name: "cortex_overrides_last_reload_successful",
-			Help: "Whether the last config reload attempt was successful.",
+			Name: "cortex_runtime_config_last_reload_successful",
+			Help: "Whether the last runtime-config reload attempt was successful.",
 		}),
+		configInfo: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cortex_runtime_config_info",
+			Help: "Information about the active runtime config.",
+		}, []string{"hash"}),
 	}
 
 	if registerer != nil {
 		registerer.MustRegister(mgr.configLoadSuccess)
+		registerer.MustRegister(mgr.configInfo)
```

Comment:
Since you're touching this code, can we switch to using `promauto.With(registerer)...`? We're trying to do that everywhere in Cortex.